src/maptm/src/maptm_ovsdb.c

```diff
@@ -128,44 +128,31 @@ bool maptm_persistent(void)
     char mapt_support[10];
 
     // Open persistent storage in read-write mode
-    ps = osp_ps_open("MAPT_SUPPORT", OSP_PS_RDWR | OSP_PS_PRESERVE);
+    ps = osp_ps_open(MAPT_SUPPORT_STORE, OSP_PS_RDWR | OSP_PS_PRESERVE);
     if (!ps)
     {
         LOGE("Failed when opening persistent storage");
         goto out;
     }
 
-    // Check if persistent storage exists
-    size = osp_ps_get(ps, "MAPT_SUPPORT", NULL, 0);
-    if (size < 0)
-    {
-        LOGE("Persistent storage does not exist");
-        goto out;
-    }
-    else if (size > 0)
-    {
-        ret = true;
-    }
-
     // Get persistent storage value
-    size = osp_ps_get(ps, "MAPT_SUPPORT", mapt_support, strlen(mapt_support));
+    size = osp_ps_get(ps, MAPT_SUPPORT_VALUE, mapt_support, strlen(mapt_support)+1);
```

Comment:
```suggestion
    size = osp_ps_get(ps, MAPT_SUPPORT_VALUE, mapt_support, sizeof(mapt_support));
```

This was already correct at some point. Once more, strlen+1 when writing, sizeof when reading.